README.md

```diff
@@ -14,13 +14,19 @@ and collating observations of distant sites and processes that are visible to us
 ## Steps performed by the Observatory
 
 1. Fetch the Sitemap XML for a news site
-2. Hit the [Google Custom Search Site Restricted JSON API](https://developers.google.com/custom-search/v1/site_restricted_api)
+2. Hit the [Discovery Engine API (service is named Google Vertex Agent Builder)](https://cloud.google.com/generative-ai-app-builder/docs/reference/rest)
    to check if the content listed is available in Google search.
-   [API Consumption](https://console.cloud.google.com/apis/api/customsearch.googleapis.com/metrics?project=ophan-reborn-2017) &
+   [API Consumption](https://console.cloud.google.com/gen-app-builder/monitoring?inv=1&invt=AbigZA&project=ophan-reborn-2017) &
```

Comment:
This page: 

https://console.cloud.google.com/gen-app-builder/monitoring?invt=AbjH8Q&project=ophan-reborn-2017

![image](https://github.com/user-attachments/assets/5ef701ea-b3d3-4b4c-8951-1b043d294170)

...has a `VIEW API METRICS` link off to this 'Discovery Engine API' page:

https://console.cloud.google.com/apis/api/discoveryengine.googleapis.com/metrics?inv=1&invt=AbjH5g&project=ophan-reborn-2017

![image](https://github.com/user-attachments/assets/fdb47a2a-5302-4f66-83e0-ea4b028fa7e6)

...which I guess is measuring similar stuff. Surprisingly though, the 'Cost' tab shows a blank graph - are we within some free quote allowance or something? If the cost is tracked somewhere else, it would be good to know!

https://console.cloud.google.com/apis/api/discoveryengine.googleapis.com/cost?inv=1&invt=AbjH5g&project=ophan-reborn-2017
![image](https://github.com/user-attachments/assets/18a4d308-2aff-4d78-b6b4-df651512115b)

Reply:
Hmm, not sure, will monitor after releasing